src/system/UsersModule/Helper/MailHelper.php

```diff
@@ -113,7 +113,7 @@ public function createAndSendRegistrationMail(UserEntity $userEntity, $userNotif
             $notificationEmail = $this->variableApi->get('ZikulaUsersModule', UsersConstant::MODVAR_REGISTRATION_ADMIN_NOTIFICATION_EMAIL, '');
             if (!empty($notificationEmail)) {
                 $authMapping = $this->authenticationMappingRepository->getByZikulaId($userEntity->getUid());
-                $rendererArgs['isVerified'] = $authMapping->isVerifiedEmail();
+                $rendererArgs['isVerified'] = null === $authMapping || $authMapping->isVerifiedEmail();
```

Comment:
This doesn't get the email, but the information about whether the email has been verified. This is not part of the user entity anymore.